src/apps/concepts/pages/CreateOrEditConceptPage.tsx

```diff
@@ -143,7 +143,7 @@ const CreateOrEditConceptPage: React.FC<Props> = ({
       title={
         context === CONTEXT.edit
           ? "Edit " +
-            startCase(toLower(concept ? concept.display_name : "concept"))
+            ( concept ? concept.display_name : "concept" )
```

Comment:
Did you try testing the changes with a working OCL instance ,cc @nk183

Reply:
Yes @HerbertYiga  it working fine
![image](https://user-images.githubusercontent.com/58779460/111427114-9d6d8800-871b-11eb-939f-5aca5520af30.png)